src/Bicep.Core.IntegrationTests/LambdaTests.cs

```diff
@@ -89,7 +89,8 @@ public void Map_function_preserves_types_accurately_integers()
             var (file, cursors) = ParserHelper.GetFileWithCursors(@"
```

Comment:
Making this configurable per test was a request that came up during the discussions meeting, as you can't currently use logical or tokens in cursor tests. The idea was that not including a default forces the test writer to choose a character that does not appear in the test; this would also help by making cursor ambiguity a caller error rather than incorrect test behavior.

There were specific objections to every single character token on the US qwerty layout, which is why I ended up using `ǂ` in hover and completion tests with type unions (though any single UCS-2 character that doesn't appear on the keyboard would be fine).